general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -193,12 +200,19 @@ def create_job(
             "target": target,
             "shots": int(repetitions),
         }
-
         if method is not None:
             json_dict["method"] = method
+        if kwargs or self._client_kwargs:
+            json_dict["options"] = json.dumps({**kwargs, **self._client_kwargs})
+            if "cq_token" in json_dict["options"]:
+                warnings.warn(
+                    "Starting with client-superstaq v0.4.20, passing in `cq_token` when "
```

Comment:
would this work?
```suggestion
                    f"Starting with {self.client_name} v0.4.20, passing in `cq_token` when "
```
also out of curiosity why are we deprecating this? `create_job` is going to continue to take kwargs, so i'm not sure i see why we'd block cq_token specifically

Reply:
We want to discourage Hilbert team from passing in `cq_token`s anywhere else but in service/provider. We can remove this deprecation notice and just add a `best practices` section in our docs.

Reply:
ah ok i understand - either way seems fine to me then

Reply:
Removed deprecation warning. Just confirmed that no one is currently passing `cq_token` anyway. Will communicate the new way just in case they need it.